app/api/resources/mentorship_relation.py

```diff
@@ -0,0 +1,173 @@
+from app.utils.ms_constants import DEFAULT_PAGE, DEFAULT_USERS_PER_PAGE
+# Namespace Models
+from app.api.models.mentorship_relation import *
+# Databases Models
+from app.database.models.bit_schema.user_extension import UserExtensionModel
+from app.database.models.ms_schema.mentorship_relation import MentorshipRelationModel
+from app.database.models.bit_schema.organization import OrganizationModel
+from app.database.models.bit_schema.program import ProgramModel
+# DAOs
+from app.api.dao.user_extension import UserExtensionDAO
+from app.api.dao.personal_background import PersonalBackgroundDAO
+from app.api.dao.mentorship_relation_extension import MentorshipRelationExtensionDAO
+from app.api.dao.organization import OrganizationDAO
+from app.api.dao.program import ProgramDAO
+
+mentorship_relation_ns = Namespace(
+    "Mentorship Relation",
+    description="Operations related to " "mentorship relations " "between users",
+)
+add_models_to_namespace(mentorship_relation_ns)
+
+mentorshipRelationExtensionDAO = MentorshipRelationExtensionDAO()
+userExtensionDAO = UserExtensionDAO()
+OrganizationDAO = OrganizationDAO()
+ProgramDAO = ProgramDAO()
+
+@mentorship_relation_ns.route("organizations/<int:organization_id>/programs/<int:program_id>/send_request")
+class SendRequest(Resource):
+    @classmethod
+    @mentorship_relation_ns.doc("send_request")
+    @mentorship_relation_ns.expect(auth_header_parser, send_mentorship_extension_request_body)
+    @mentorship_relation_ns.response(
+        HTTPStatus.CREATED, "%s" % messages.MENTORSHIP_RELATION_WAS_SENT_SUCCESSFULLY
+    )
+    @mentorship_relation_ns.response(
+        HTTPStatus.BAD_REQUEST,
+        "%s\n%s\n%s\n%s\n%s\n%s\n%s\n%s\n%s\n%s\n%s\n%s\n%s\n%s\n%s"
+        % (
+            messages.NO_DATA_WAS_SENT,
+            messages.MATCH_EITHER_MENTOR_OR_MENTEE,
+            messages.MENTOR_ID_SAME_AS_MENTEE_ID,
+            messages.END_TIME_BEFORE_PRESENT,
+            messages.MENTOR_TIME_GREATER_THAN_MAX_TIME,
+            messages.MENTOR_TIME_LESS_THAN_MIN_TIME,
+            messages.MENTOR_NOT_AVAILABLE_TO_MENTOR,
+            messages.MENTEE_NOT_AVAIL_TO_BE_MENTORED,
+            messages.MENTOR_ALREADY_IN_A_RELATION,
+            messages.MENTEE_ALREADY_IN_A_RELATION,
+            messages.MENTOR_ID_FIELD_IS_MISSING,
+            messages.MENTEE_ID_FIELD_IS_MISSING,
+            messages.NOTES_FIELD_IS_MISSING,
+            messages.ORGANIZATION_DOES_NOT_EXIST,
+            messages.PROGRAM_DOES_NOT_EXIST,
+        ),
+    )
+    @mentorship_relation_ns.response(
+        HTTPStatus.UNAUTHORIZED,
+        "%s\n%s\n%s"
+        % (
+            messages.TOKEN_HAS_EXPIRED,
+            messages.TOKEN_IS_INVALID,
+            messages.AUTHORISATION_TOKEN_IS_MISSING,
+        ),
+    )
+    @mentorship_relation_ns.response(
+        HTTPStatus.NOT_FOUND,
+        "%s\n%s" % (messages.MENTOR_DOES_NOT_EXIST, messages.MENTEE_DOES_NOT_EXIST),
+    )
+    def post(cls,organization_id,program_id):
```

Comment:
A critical point on how you approach this issue. Here I can see that you try to use the same logic implemented in Mentorship System backend `app/api/resources/mentorship_relation.py` on creating mentorship relation. This is not how we want to integrate the MS APIs in BIT. Instead, you should send internal api request to MS backend to create the new mentorship relation on MS side, then use the new mentorship relation instance id to create the mentorship relation extension instance. 
An example of this internal call can be seen on app/api/resources/user.py. Please check [my explanation](https://github.com/anitab-org/bridge-in-tech-backend/issues/161#issuecomment-731728787) to @vj-codes on the approach I made on db_add_mock.py on this feature. Feel free to ask anything if you're unsure.